fastlane/Fastfile

```diff
@@ -54,5 +55,63 @@ platform :android do
       message: 'Update translations',
       allow_nothing_to_commit: true
     )
+
+    desc 'Builds a prototype build and uploads it to S3'
+    lane :build_and_upload_prototype_build do
+      unless ENV['BUILDKITE_ARTIFACTS_S3_BUCKET']
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/Gravatar-SDK-Android/blob/a5e3bf10b0a8bbe71470202d5a27bdcf3c1ae6a2/fastlane/Fastfile#L61"></span>Style/IfUnlessModifier: Favor modifier <code>unless</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>